nvim/stdio_stream.lua

```diff
@@ -0,0 +1,39 @@
+local uv = require('luv')
+
+
+local StdioStream = {}
+StdioStream.__index = StdioStream
+
+function StdioStream.open()
```

Comment:
I can't remember, but is `local` needed here? Might be good to make it explicit, even if not required.